portfolio/src/main/webapp/script.js

```diff
@@ -381,3 +381,31 @@ function postMarker(lat, lng, description) {
 
   fetch('/your-map-data', {method: 'POST', body: params});
 }
+
+google.charts.load('current', {'packages':['timeline']});
+google.charts.setOnLoadCallback(drawChart);
+
+/** Creates a chart and adds it to the page. */
+function drawChart() {
+  var container = document.getElementById('chart-container');
+  var chart = new google.visualization.Timeline(container);
+  var dataTable = new google.visualization.DataTable();
+
+  dataTable.addColumn({ type: 'string', id: 'ProjectNumber' });
+  dataTable.addColumn({ type: 'string', id: 'ProjectName' });
+  dataTable.addColumn({ type: 'date', id: 'Start' });
+  dataTable.addColumn({ type: 'date', id: 'End' });
+  dataTable.addRows([
+    [ '1', 'Movie Kiosk Software',          new Date(2018, 2, 1),  new Date(2018, 5, 1) ],
+    [ '2', 'Directed Graph Structure',      new Date(2019, 2, 1),  new Date(2019, 5, 1) ],
+    [ '3', 'E-shop - Computer devices',     new Date(2019, 6, 1),  new Date(2019, 9, 1) ],
+    [ '4', 'Electrical Forecast Database',  new Date(2020, 2, 1),  new Date(2020, 5, 1) ],
+    [ '5', 'E-Library System',              new Date(2020, 6, 1),  new Date(2020, 9, 1) ],
+    [ '6', 'IOU - favour exchange',         new Date(2020, 6, 1),  new Date(2020, 9, 1) ]]);
```

Comment:
it might be nice to make this a constant since it's something that's hard coded and never edited.
https://google.github.io/styleguide/jsguide.html#naming-constant-names (you're not using closure so don't worry about the @const)